### DISCUSSION TITLE
BPeripherals

### DISCUSSION THREAD

**Discussion by SpuSpu** [NillerMedDild/Enigmatica6#1463] [Ideas]
I recently found out that Peripherals for computer craft: tweaked has made an appearance for 1.16.5. Is there any plans for the future to add this mod (https://www.curseforge.com/minecraft/mc-mods/bperipherals)? I have not tested it out myself but I know a little about old computer craft and would love to have this in the pack, if it works.

**Comment by NielsPilgaard** [comment]
At the moment it's completely useless in Enigmatica 6, unless you want a mod that makes a database :P

**Reply by SpuSpu** [reply]
Welp thats embarrassing, I hope it gets to be updated to be like the old one.

**Reply by merlerm** [reply]
Looking at the curse page, the FE monitor looks interesting, I could think of some use cases for it in the pack. Would need to test ingame since it's labelled as WIP but maybe we should keep an eye on it
